tests/std/tests/P0896R4_views_istream/test.cpp

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <cstddef>
+#include <ranges>
+#include <sstream>
+
+#include <range_algorithm_support.hpp>
+
+using namespace std;
+
+static constexpr int expected_empty[]  = {-1, -1, -1, -1, -1};

```

Comment:
`static` is extraneous for `constexpr` variables at namespace scope. (Also on 15.)

Reply:
:+1: